packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -599,11 +605,74 @@ abstract class ScrollView extends StatelessWidget {
 ///    the scroll position without using a [ScrollController].
 ///  * [IndexedSemantics], which allows annotating child lists with an index
 ///    for scroll announcements.
-class CustomScrollView extends ScrollView {
+///
+class CustomScrollView extends StatelessWidget {
```

Comment:
Why this change?

Reply:
CustomScrollView should not get a Scrollbar automatically, but with this approach of copying ScrollBehaviors to modify for individual widgets, we need to be able to get the current ScrollBehavior in order to copy it.